File: specification/resourceconnector/resource-manager/Microsoft.ResourceConnector/stable/2022-10-27/examples-java/AppliancesListClusterUserCredential.java
/** Samples for Appliances ListClusterUserCredential. */
public final class Main {
    /*
     * x-ms-original-file: specification/resourceconnector/resource-manager/Microsoft.ResourceConnector/stable/2022-10-27/examples/AppliancesListClusterUserCredential.json
     */
    /**
     * Sample code: ListClusterUserCredentialAppliance.
     *
     * @param manager Entry point to ResourceConnectorManager.
     */
    public static void listClusterUserCredentialAppliance(
        com.azure.resourcemanager.resourceconnector.ResourceConnectorManager manager) {
        manager
            .appliances()
            .listClusterUserCredentialWithResponse(
                "testresourcegroup", "appliance01", com.azure.core.util.Context.NONE);
    }
}

File: specification/resourceconnector/resource-manager/Microsoft.ResourceConnector/stable/2022-10-27/examples-js/AppliancesListClusterUserCredential.js
const { ResourceConnectorManagementClient } = require("@azure/arm-resourceconnector");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Returns the cluster user credentials for the dedicated appliance.
 *
 * @summary Returns the cluster user credentials for the dedicated appliance.
 * x-ms-original-file: specification/resourceconnector/resource-manager/Microsoft.ResourceConnector/stable/2022-10-27/examples/AppliancesListClusterUserCredential.json
 */
async function listClusterUserCredentialAppliance() {
  const subscriptionId =
    process.env["RESOURCECONNECTOR_SUBSCRIPTION_ID"] || "11111111-2222-3333-4444-555555555555";
  const resourceGroupName = process.env["RESOURCECONNECTOR_RESOURCE_GROUP"] || "testresourcegroup";
  const resourceName = "appliance01";
  const credential = new DefaultAzureCredential();
  const client = new ResourceConnectorManagementClient(credential, subscriptionId);
  const result = await client.appliances.listClusterUserCredential(resourceGroupName, resourceName);
  console.log(result);
}
